src/styles/page-template/index.md.njk

```diff
@@ -98,6 +98,13 @@ Variables can be set with:
       <td class="govuk-table__cell">Add a class to the <code>&lt;html&gt;</code> element.</td>
     </tr>
 
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">pageTitleLang</td>
+      <td class="govuk-table__cell">
+        Set the language of the page title. This is useful if some of the page elements, such as navigation and footer, are in English whereas the title and page content are in a different language. It is not necessary to set this if it matches <code>htmlLang</code>.
```

Comment:
I've had a go at scrunching this up so it's more concise (as it's good to avoid long table cells).

```suggestion
        Set the language of the page title if it's different to <code>htmlLang</code>.
```

Reply:
Thanks @m-green. Your suggestion is definitely more concise - however I wonder if we could somehow incorporate some version of "Useful if some of the page elements, such as navigation and footer, are in English whereas the title and page content are in a different language." It's a bit of a weird scenario and not really obvious so it would be good to explain that.
 

Reply:
Ah sorry, I think I misunderstood.
In the example you mentioned, wouldn't `pageTitleLang` and `htmlLang' both be set to the same - because the title and page content are both the same language?

Reply:
New suggestion after our chat:
Set the language of the `<title>` element if it's different to <code>htmlLang</code>.